### PR TITLE
add localhgt

### PR DESCRIPTION
**Pull request by wshuai294**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by wshuai294** [comment]
@bioconda/core   Hello, I want to publish LocalHGT, which is a method to detect HGT events from the microbiome. Now, it passed all the checks. Could you please help me review and merge it?

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

File: recipes/localhgt/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Ensure we run successfully using either conda-forge or defaults ncurses
+# (unlike other platforms, the latter does not automatically pull in libtinfo)
```
Comment (martin-g) [inline recipes/localhgt/build.sh:4]: The comment above is not very clear to me. How ncurses (a TUI library) is related here ?

File: recipes/localhgt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("localhgt", max_pin="x") }}
+  entry_points:
+    - localhgt = localhgt:main
+
+source:
+  url: https://github.com/deepomicslab/LocalHGT/archive/refs/tags/v{{ version }}.tar.gz
+  md5: 8438fdd9cb3ad02fe331a09d4ca0ccfe
```
Comment (martin-g) [inline recipes/localhgt/meta.yaml:16]: Old recipes use `md5` but `sha256` is recommended for newer recipes.

File: recipes/localhgt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  license: MIT
+  summary: Tools for dealing with SAM, BAM and CRAM files
+
+test:
+  commands:
+    - localhgt --help
+
+extra:
+  identifiers:
+    - biotools:localhgt
```
Comment (martin-g) [inline recipes/localhgt/meta.yaml:50]:
```suggestion
    - biotools:localhgt
  recipe-maintainers:
    - wshuai294
```

File: recipes/localhgt/build.sh (hunk 1 above)
Comment (wshuai294) [reply recipes/localhgt/build.sh]: Hi, thanks for the review. The comment is nonsense. So, I delete them.

File: recipes/localhgt/meta.yaml (hunk 2 above)
Comment (wshuai294) [reply recipes/localhgt/meta.yaml]: I revised this as suggested.

File: recipes/localhgt/meta.yaml (hunk 3 above)
Comment (wshuai294) [reply recipes/localhgt/meta.yaml:50]: I revised this as suggested.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @wshuai294

**Pull request merged**